components/library/botan/Makefile

```diff
@@ -47,24 +61,14 @@ CONFIGURE_OPTIONS+= --includedir=/usr/include
 CONFIGURE_OPTIONS+= --cc gcc
 CONFIGURE_OPTIONS+= --cc-bin $(CXX)
 CONFIGURE_OPTIONS+= --os sunos
-CONFIGURE_OPTIONS+= --enable-sse2
-CONFIGURE_OPTIONS+= --enable-altivec
-CONFIGURE_OPTIONS+= --enable-movbe
-CONFIGURE_OPTIONS+= --with-openssl
-CONFIGURE_OPTIONS+= --disable-modules=ecdsa
 CONFIGURE_OPTIONS+= --with-bzip2
 CONFIGURE_OPTIONS+= --with-zlib
 
-COMPONENT_POST_INSTALL_ACTION= \
-            (cd $(PROTOUSRDIR) ;  \
-             $(MV)  include/botan-1.10/botan/build.h include/botan-1.10/botan/build-$(BITS).h )
-
 # Auto-generated dependencies
 REQUIRED_PACKAGES += $(GCC_RUNTIME_PKG)
 REQUIRED_PACKAGES += $(GXX_RUNTIME_PKG)
 REQUIRED_PACKAGES += compress/bzip2
-REQUIRED_PACKAGES += library/security/openssl
 REQUIRED_PACKAGES += library/zlib
-REQUIRED_PACKAGES += shell/ksh93
 REQUIRED_PACKAGES += system/library
 REQUIRED_PACKAGES += system/library/math
+REQUIRED_PACKAGES += runtime/python-$(subst .,,$(PYTHON_VERSION))
```

Comment:
no, it was my attempt to resolve the pyc dependency, but basically correct

Reply:
> no, it was my attempt to resolve the pyc dependency, but basically correct

If you run `gmake REQUIRED_PACKAGES` again, will it be replaced by runtime/python-XX (where XX is 27, 35, 37, or 39), or simply removed?  If removed, you need to move this up (above the `Auto-generated dependencies` comment).  If replaced, then you could leave it as is, but better would be to replace it,  because it will get replaced anyway later.

Reply:
gmake REQUIRED_PACKAGES will add

+REQUIRED_PACKAGES += runtime/python-39

so I will choose this too.